src/io/parquet/mod.rs

```diff
@@ -9,18 +9,23 @@
 //! use geoarrow::io::parquet::ParquetReaderOptions;
 //! use std::fs::File;
 //!
+//! # #[cfg(feature = "parquet_compression")]
+//! # {
```

Comment:
I'm curious, why do you have a `{` here? Is that needed?

Reply:
It's to feature-gate the whole test in the a block.

Reply:
oh that makes sense